robowflex_library/src/io/visualization.cpp

```diff
@@ -218,9 +219,10 @@ void IO::RVIZHelper::addGoalMarker(const std::string &name, const MotionRequestB
                         continue;
 
                     auto q = TF::quaternionMsgToEigen(og.orientation);
+                    Eigen::Affine3d qframe(Eigen::Translation3d(frame.translation()));
```

Comment:
Silly request, but some inline comments might help here; it's not immediately clear what you're doing by looking at just the translation of `frame`.